components/x-privacy-manager/src/privacy-manager.jsx

```diff
@@ -0,0 +1,115 @@
+/**
+ * @typedef {{ ok: boolean, status?: number }} _Response
+ */
+
+import { h } from '@financial-times/x-engine';
+import { withActions } from '@financial-times/x-interaction';
+
+import s from './privacy-manager.scss';
+import { RadioBtn } from './radio-btn';
+import { LoadingMessage, ResponseMessage } from './messages';
+
+// TODO: Replace with the genuine endpoint
+export const CONSENT_API = 'https://consent.ft.com';
+
+export const withCustomActions = withActions(() => ({
+	onConsentChange() {
+		return ({ consent = true }) => ({ consent: !consent });
+	},
+
+	sendConsent() {
+		return async ({ consent }) => {
+			const body = JSON.stringify({
+				demographic: consent,
+				behavioural: consent,
+				programmatic: consent
+			});
+
+			try {
+				const res = await fetch(CONSENT_API, { method: 'PATCH', body });
+				return { _response: { ok: res.ok } };
+			} catch (err) {
+				return { _response: { ok: false } };
+			}
+		};
+	}
+}));
+
+/**
+ * @param {boolean} isLoading
+ * @param {_Response} response
+ * @param {string} referrer
+ */
+function renderMessage(isLoading, response, referrer) {
+	if (isLoading) return <LoadingMessage />;
+	if (response) return <ResponseMessage success={response.ok} referrer={referrer} />;
+	return null;
+}
+
+/**
+ * @param {{
+ *   consent?: boolean
+ *   referrer?: string
+ *   legislation?: string[]
+ *   actions: {
+ *     onConsentChange: () => void
+ * 	   sendConsent: () => Promise<{_response: _Response }>
+ *   },
+ *   isLoading: boolean
+ *   _response: _Response | undefined
+ * }} Props
+ */
+export function BasePrivacyManager({
+	consent = true,
+	referrer,
+	actions,
+	isLoading,
+	_response = undefined
+}) {
+	return (
+		<div className={s.consent}>
+			<h1 className={s.consent__title}>Do Not Sell My Personal Information</h1>
```

Comment:
I'm making an assumption that given this is a component that it may be within a page itself. Is there any risk that a h1 will already be on the page?

Reply:
Great question, but in this instance the component is the entirety of the page and the h1's landmarking is accurate here. 
Since this marks our first steps towards a unified Privacy module, and lots of further evolution, it's definitely a point to keep in mind!